bench/meili/mapmatch.cc

```diff
@@ -0,0 +1,117 @@
+#include <iostream>
+#include <sstream>
+
+#include <benchmark/benchmark.h>
+#include <boost/property_tree/ptree.hpp>
+#include <rapidjson/rapidjson.h>
+
+#include "meili/map_matcher_factory.h"
+#include "meili/measurement.h"
+#include "sif/costconstants.h"
+#include "sif/costfactory.h"
+#include "tyr/actor.h"
+
+using namespace valhalla::midgard;
+using namespace valhalla::meili;
+using namespace valhalla::sif;
+
+namespace {
+
+#if !defined(VALHALLA_SOURCE_DIR)
```

Comment:
super minor nit: if its a single define being tested, you can use the less verbose form `ifndef` too. And its totally fine to ignore this nit.